common.tf

```diff
@@ -14,6 +14,11 @@ resource "aws_secretsmanager_secret" "preservica_secret" {
   name = "${local.environment}-preservica-api-login-details-${random_string.preservica_user.result}"
 }
 
+resource "aws_secretsmanager_secret" "slack_webhook_url" {
```

Comment:
Sorry to be a pain but can you put this in SSM parameter store instead of secrets manager. 

Parameter store is a lot cheaper and just as secure if you use SecureString. We've already got quite a bit in parameter store as well. 

Secrets manager makes sense for preservica because we can change the credentials programatically so the rotation features are worth it but the slack webhook will be updated manually, if at all. 

You can add it as a resource, there is a module but that's more for creating lots of parameters at once.

```
resource "aws_ssm_parameter" "slack_webhook" {
  name = "/${local.environment}/slack/webhook"
  type = "SecureString"
}
```

You'll have to change your lambda code to get it from SSM instead

Reply:
Sure. Sorry, I misread; thought the card said "Secrets Manager"🤦